prow/add-ist

```diff
@@ -56,7 +56,7 @@ def add_tag(imagestream: dict, tag_name: str) -> None:
 
 
 stream = None
-with open("overlays/cnv-prod/imagestreamtags.yaml") as ist:
+with open("overlays/PROD/imagestreamtags.yaml") as ist:
```

Comment:
in the prow folder, i only see overlays/smaug in your branch, not overlays/PROD

Reply:
thanks, fixing it :+1: